ui/mage/presets.ts

```diff
@@ -1134,4 +1149,245 @@ export const FFB_P2_PRESET = {
 		  ]
 		}
 	  ]}`),
+};
+
+
+export const FIRE_P3_PRESET_HORDE = {
+	name: 'Fire P3 Preset Horde',
+	enableWhen: (player: Player<Spec.SpecBalanceDruid>) => player.getFaction() == Faction.Horde,
```

Comment:
This condition needs to also check for the player spec, e.g. `player.getTalentTree() == 1`. We don't want to show fire presets when the user has arcane settings, for example.

Same comment for the alliance preset.

Reply:
ok. fixed it